search/search_integration_tests/generate_tests.cpp

```diff
@@ -32,6 +34,8 @@ void MakeFeature(TestMwmBuilder & builder, pair<string, string> const & tag, m2:
   FeatureBuilder1 fb;
   fb.SetParams(params);
   fb.SetCenter(pt);
+  fb.GetMetadataForTesting().Set(feature::Metadata::FMD_TEST_ID, strings::to_string(g_lastId));
+  ++g_lastId;
```

Comment:
nit: g_lastId should be reset in the beginning of every test for reproducibility wrt filters/order of tests.

Reply:
Easy peasy.